### PR TITLE
Include the error time in the report payload

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: src/Report.php [hunk 1]
```diff
@@ -155,6 +162,7 @@ public static function fromNamedError(Configuration $config, $name, $message = n
     protected function __construct(Configuration $config)
     {
         $this->config = $config;
+        $this->time = gmdate('Y-m-d\TH:i:s\Z');
```
Comment (GrahamCampbell) [inline src/Report.php]: Do we need an integer timestamp here, or a formatted timestamp?

Comment (kattrali) [inline src/Report.php]: Formatted timestamp

Comment (GrahamCampbell) [inline src/Report.php]: Ok, I asked after seeing the event worker code treating it as an integer.

**Comment by kattrali** [comment]
⭐ 🚢

**Pull request merged**